site/_docs/reference.md

```diff
@@ -1158,8 +1158,8 @@ name will have been converted to upper case also.
 | BIGINT      | 8 byte signed integer     | Range is -9223372036854775808 to 9223372036854775807
 | DECIMAL(p, s) | Fixed point             | Example: 123.45 or DECIMAL '123.45' is a DECIMAL(5, 2) value.
 | NUMERIC     | Fixed point               |
-| REAL, FLOAT | 4 byte floating point     | 6 decimal digits precision
-| DOUBLE      | 8 byte floating point     | 15 decimal digits precision
+| REAL, FLOAT | 4 byte floating point     | 6 decimal digits precision.  Examples: 1.2E2, CAST('Infinity' AS REAL), CAST('-Infinity' AS FLOAT), CAST('NaN' AS FLOAT)
```

Comment:
FLOAT is actually the same as DOUBLE. (It's complicated, but standard SQL allows variable-precision floating point, and but we only allow 8 bytes.) I think you should put FLOAT on a separate line saying 'Equivalent to DOUBLE'.

No '.' at end of line.